docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,65 @@
+> [!NOTE]
+> This topic is open for collaboration on a separate branch
+
+# Overview
+
+The Execution Layer focuses exclusively on executing the state transition function (STF). This role encompasses addressing two primary questions[¹]:
```

Comment:
I would rephrase this. Maybe it makes more sense within EELS but talking about execution layer in general, it does much more than that (keeping the history, communication api for users,..)